docs/docs/deployment/hosting/locally-api.md

```diff
@@ -246,6 +246,10 @@ the below variables will be ignored.
   ignored.
 - `DISABLE_FLAGSMITH_UI`: Disable the Flagsmith UI which can be rendered by the API containers in a single container
   environment. Use `True` to disable, defaults to `False`.
+- `SEGMENT_CONDITION_VALUE_LIMIT`: Configure the size of the segment condition value in bytes. Default is 1000.
```

Comment:
Perhaps we should add a warning that going above the default will make the data inoperable with the SaaS offering since we won't be able to import their data? This is kinda locking them in to hosting themselves, no?

Reply:
Great catch - yes I'll do that!